File: Baekjoon/백준1769Multiple.java
import java.util.Scanner;

public class 백준1769Multiple {

	public static void main(String[] args) {
		Scanner input=new Scanner(System.in);
		
		int X=input.nextInt();
		String Xtemp=Integer.toString(X);
		
		int result=0;
		int turn=0;
		
		while(Xtemp.length()<8) {
			if(Xtemp.length()==1) {
				result+=Integer.parseInt(Xtemp);
			}else {
				turn++;
				for(int i=0; i<Xtemp.length(); i++) {
					result+=Xtemp.charAt(i)-'0';
				}
				Xtemp=Integer.toString(result);
			}
			
			if(result<10) {
				System.out.println(turn);
				if(result%3 ==0) {
					System.out.println("YES");
				}else {
					System.out.println("No");
				}
				break;
			}
			result=0;
		}
	}
}

File: Baekjoon/백준1769Multiple.py
if __name__=='__main__' :
    x=list(map(int,input()))
    count=0

    while(len(x)!=1):
        count+=1
        temp=str(sum(x))
        x=list(map(int,temp))

    result="YES"
    if x[0]%3 !=0:
        result="NO"

    print(count)
    print(result)